icekit/workflow/models.py

```diff
@@ -69,6 +69,14 @@ def __str__(self):
 
 class WorkflowStateMixin(models.Model):
     workflow_states = GenericRelation(WorkflowState)
+    brief = models.TextField(
+        blank=True,
+        help_text="A document brief describing the purpose of this item"
+    )
+    admin_notes = models.TextField(
```

Comment:
Now this is here, there are two `admin_notes` fields available in the collections models. If it's not trivial to migrate the collections field away, could you make a new ticket for it please?